src/Compilers/CSharp/Portable/BoundTree/BoundNode.cs

```diff
@@ -18,24 +18,29 @@ internal abstract partial class BoundNode
         public readonly SyntaxNode Syntax;
 
         [Flags()]
-        private enum BoundNodeAttributes : byte
+        private enum BoundNodeAttributes : ushort
         {
             HasErrors = 1 << 0,
             CompilerGenerated = 1 << 1,
-            TopLevelNullableUnset = 0,
-            TopLevelNullable = 1 << 2,
-            TopLevelNonNullable = 1 << 3,
-            TopLevelUnknown = TopLevelNullable | TopLevelNonNullable,
-            TopLevelNullabilityMask = TopLevelUnknown,
+            IsSuppressed = 1 << 2,
+
+            // Bit 3: Whether nullability has been set
+            // Bit 4: 1 if the node is nullable, 0 if the node is not nullable
```

Comment:
>// Bit 4: 1 if the node is nullable, 0 if the node is not nullable [](start = 12, length = 66)

nit: Perhaps "// Bit 4: 1 if the node has maybe-null state, 0 if the node is not-null"